file-formats/apob/type/zif_aff_apob_v1.intf.abap

```diff
@@ -0,0 +1,62 @@
+INTERFACE zif_aff_apob_v1
+  PUBLIC.
+  TYPES:
+    "! <p class="shorttext">Application Object Description Type</p>
+    "! Attributes of Application Object Description Type
+    BEGIN OF ty_apob_text,
+      "! <p class="shorttext">Language Code</p>
+      "! Language Code
+      "! $required
+      lang    TYPE c LENGTH 1,
+      "! <p class="shorttext">Application Object</p>
+      "! Application Object Name
+      "! $required
+      ao_name TYPE c LENGTH 30,
+      "! <p class="shorttext">Application Object Description</p>
+      "! Application Object Description
+      ao_desc TYPE c LENGTH 80,
+    END OF ty_apob_text,
+
+    "! <p class="shorttext">Application Object Description Table Type</p>
+    "! Table of all Application Object Description (may be empty)
+    ty_app_object_description TYPE SORTED TABLE OF ty_apob_text WITH UNIQUE KEY lang ao_name.
+
+
+
+  TYPES:
+    "! <p class="shorttext">Application Object</p>
+    "! Application Object
+    BEGIN OF ty_main,
+      "! <p class="shorttext">Format Version</p>
+      "! Format version
+      "! $required
+      format_version                 TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header                         TYPE zif_aff_types_v1=>ty_header_60,
+      "! <p class="shorttext">Application Object</p>
+      "! Application Object Name
+      "! $required
+      ao_name                        TYPE c LENGTH 30,
```

Comment:
Is this the name of the object itself? Then this can be removed.

Reply:
removed the field